wiki/en/FAQ.md

```diff
@@ -28,9 +28,9 @@ Bear in mind also that all participants should follow [The Golden Rule](/wiki/Cl
 
 ### How do I know if I can join a server? Are there rules?
 
-In general, if somebody sets up a public server, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. However, some servers may state their policies in the welcome message you will see in the chat window.
+In general, if somebody sets up a Registered Server, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. However, some servers may state their policies in the welcome message you will see in the chat window.
```

Comment:
> if somebody sets up a Registered Server, they accept that anyone can play on it

Not quite right.  You can register a Server with a Directory and still have it Private - i.e. server firewall only allows certain client connections, Directory firewall only allows certain client connections and Directory only allows certain Server registrations.

If you Register a Server, you accept that anyone with access to the Directory can see that your server is registered.